cmd/lncli/walletrpc_active.go

```diff
@@ -747,6 +762,20 @@ func fundPsbt(ctx *cli.Context) error {
 		}
 	}
 
+	if ctx.IsSet("change_type") {
+		switch addressType := ctx.String("change_type"); addressType {
+		case "p2tr":
+			req.ChangeType = walletrpc.ChangeAddressType_CHANGE_ADDRESS_TYPE_P2TR //nolint:lll
```

Comment:
I think you can add the `//nolint:lll` directly before the offending line to not make that line longer? So between the `case "p2tr":` and the `req.ChangeType = ...`.